internal/daemon/dpusidemanager.go

```diff
@@ -261,9 +263,20 @@ func (d *DpuSideManager) setupReconcilers() {
 		mgr, err := ctrl.NewManager(d.config, ctrl.Options{
 			Scheme: scheme.Scheme,
 			NewCache: func(config *rest.Config, opts cache.Options) (cache.Cache, error) {
+				// Watch ServiceFunctionChains only in operator namespace, but Pods in both namespaces.
+				// ServiceFunctionChain CR is namespace scoped to be tied to the dpu-operator namespace.
+				// It is not a clusterwide resource.
 				opts.DefaultNamespaces = map[string]cache.Config{
 					vars.Namespace: {},
 				}
+				opts.ByObject = map[client.Object]cache.ByObject{
+					&corev1.Pod{}: {
+						Namespaces: map[string]cache.Config{
```

Comment:
This is a bit weird

Reply:
Could you please elaborate on the weirdness?

Reply:
Now I see what you're doing. You're configuring the informer to only cache local SFCs?